src/pages/sign-on/components/desktop/ForgotPasswordHelper.tsx

```diff
@@ -52,10 +66,23 @@ export const ForgotPasswordHelper = ({
             <div>{messages.fromHeader}</div>
             <div>{messages.subjectHeader}</div>
           </div>
-          <div className={styles.emailText}>
-            <div>{messages.from}</div>
-            <div>{messages.subject}</div>
-          </div>
+          <Toast
+            text={messages.copied}
+            fillParent={false}
+            mount={MountPlacement.PARENT}
+            placement={ComponentPlacement.TOP_LEFT}
```

Comment:
wonder if this should be center, but up to julian if he has any thoughts! code LGTM!